docs/conf.py

```diff
@@ -137,20 +142,20 @@
 
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
-#html_theme_path = []
+#html_theme_path = astropy_sphinx_theme.get_html_theme_path()
```

Comment:
You added something but it is still commented. Is that intentional?

Reply:
Yup, but the addition has been removed